user-guide/docs/usecases/README.md

```diff
@@ -27,16 +31,25 @@
     | - |
     | ![forked-repo](docs/images/02-forked-repo.png) |
 
+    <a id="find-usecase">
+
 3. Navigate to your use case folder, which is located in the `user-guide/docs/usecases` folder. The folders are named after the PI, so find the folder with your name to edit your template. **Always check you are only editing your use case folder**.
```

Comment:
💡 Instead of `<a>`, I could use a heading, because GitHub will automatically assign it an ID. Headings are also a way to simplify and highlight each step.

<details><summary>Use Heading and Auto ID</summary>

Add heading. ID would become `#find-use-cases`.

```suggestion
### 3. Find Use Case

Navigate to your use case folder, which is located in the `user-guide/docs/usecases` folder. The folders are named after the PI, so find the folder with your name to edit your template. **Always check you are only editing your use case folder**.
```

</details>

<details><summary>Use Heading and Manual ID</summary>

Add heading. ID would stay `#find-usecase`.

```suggestion
### <a id="find-usecase"> 3. Find Use Case

Navigate to your use case folder, which is located in the `user-guide/docs/usecases` folder. The folders are named after the PI, so find the folder with your name to edit your template. **Always check you are only editing your use case folder**.
```

</details>